packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -3038,7 +3041,7 @@ describe('plugin-meetings', () => {
           });
 
           it('addMedia() works correctly when media is enabled with tracks to publish and track is muted', async () => {
-            fakeMicrophoneStream.muted = true;
+            fakeMicrophoneStream.userMuted = true;
```

Comment:
do we have any tests for tracks being system muted?

Reply:
I'll check this too.